features/cellular/framework/AT/AT_CellularContext.cpp

```diff
@@ -592,18 +613,27 @@ nsapi_error_t AT_CellularContext::open_data_channel()
     }
 
     _at.set_is_filehandle_usable(false);
-
+#if (DEVICE_SERIAL && DEVICE_INTERRUPTIN)
```

Comment:
Mustn't be conditional - see other comments.

Reply:
removed #ifdef